plugin/broadcastExtensionFiles/MembraneBroadcastSampleHandler.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import MembraneRTC
+import ReplayKit
+import WebRTC
+import os.log
+
+/// App Group used by the extension to exchange buffers with the target application
+let appGroup = "{{GROUP_IDENTIFIER}}"
+
+let logger = OSLog(subsystem: "{{BUNDLE_IDENTIFIER}}.MembraneBroadcastSampleHandler", category: "Broadcaster")
+
+/// An example `SampleHandler` utilizing `BroadcastSampleSource` from `MembraneRTC` sending broadcast samples and necessary notification enabling device's screencast.
```

Comment:
I know it is not your code, but name of this class should be changed to sth related to this repo, not old one. Comment above also should be updated.

Reply:
Let's update in future work, when this repo will be cleaned up